### PR TITLE
Add "refresh" capability

### PR DESCRIPTION
**Pull request by acowley**
Hitting "g" on the *dashboard* will redraw its contents, including any
updates to the recent file list.

### CODE REVIEW

File: dashboard.el [hunk 1]
```diff
@@ -54,6 +55,7 @@ Return entire list if `END' is omitted."
   :syntax-table nil
   :abbrev-table nil
   (linum-mode -1)
+  (page-break-lines-mode 1)
```
Comment (rakanalh) [inline dashboard.el]: We're already activating page-break-lines-mode here:
https://github.com/rakanalh/emacs-dashboard/pull/20/files#diff-49993283b3a3adc31101f309fab24b1cR265

File: dashboard.el [hunk 2]
```diff
@@ -245,7 +248,8 @@ Optionally, provide NO-NEXT-LINE to move the cursor forward a line."
 		(dashboard-insert-page-break)
 		))
 	    dashboard-items))
-    (dashboard-mode)))
+    (dashboard-mode)
+    (goto-char 0)))
```
Comment (rakanalh) [inline dashboard.el]: use `(point-min)`

File: dashboard.el (hunk 1 above)
Comment (acowley) [inline dashboard.el]: Yeah, it wasn't working for me before I added that line, but I'll try again to see if I had just gotten myself into a funny state.

**Comment by rakanalh** [comment]
I see both places as candidates for enabling the mode... Could you please check which one works best and remove the other?

**Comment by acowley** [comment]
Sorry I didn't get to the requested revisions yet, @rakanalh!

**Comment by rakanalh** [comment]
No worries, i fixed my comments and pushed to master.

Thanks alot!

**Pull request merged**